Source/IGListAdapter.m

```diff
@@ -319,14 +321,14 @@ - (void)performUpdatesAnimated:(BOOL)animated completion:(IGListUpdaterCompletio
     }
 
     NSArray *fromObjects = self.sectionMap.objects;
-    NSArray *newObjects = [dataSource objectsForListAdapter:self];
+    NSArray *uniqueObjects = objectsWithDuplicateIdentifiersRemoved([dataSource objectsForListAdapter:self]);
```

Comment:
If we're going to put this here (which we don't need to yet), we should remove it [from here](https://github.com/Instagram/IGListKit/blob/master/Source/IGListAdapterUpdater.m#L109) since that'll be wasteful. I lean towards not messing w/ it here since it'll be extra work (dedupe every update call instead of at coalescence).

I'm actually going to patch that change then import this!